app/elements/socobo-recipe/socobo-recipe-overview.html

```diff
@@ -508,7 +508,7 @@
           for (var e in el) {
             var listItem = el[e];
             var internalEl = Polymer.dom(listItem.root).querySelector(".check paper-checkbox");
```

Comment:
The only thing for this property is to check / uncheck The checkbox (checked={{isChecked}}).